main/autogen/autoloader.cc

```diff
@@ -498,6 +486,30 @@ void populateAutoloadTasksAndCreateDirectories(const core::GlobalState &gs, vect
 }
 }; // namespace
 
+const core::FileRef DefTree::definingFile() const {
```

Comment:
btw, there's almost no value to having a `const` by-value return type, and sometimes can actually get in the way of optimizations.

https://stackoverflow.com/questions/8406898/benefits-of-using-const-with-scalar-type-e-g-const-double-or-const-int

https://stackoverflow.com/questions/8716330/purpose-of-returning-by-const-value